integration-tests/test_stdin_to_stdout.py

```diff
@@ -21,3 +23,25 @@ def test_stdin_to_stdout(tmpdir):
     assert result.get("gender") == "M"
 
     os.remove(output_file)
+
+@pytest.mark.xfail(run=False)
+def test_stdin_to_stdout_filtered(tmpdir):
+    tested_data = '{"id": 121, "fname": "joe", "lname": "allen", "country_code": "US", "country_name": "united states", "gender": "M"}'
+    tested_data2 = '{"id": 122, "fname": "shmoe", "lname": "allen", "country_code": "US", "country_name": "united states", "gender": "M"}'
+    tested_data3 = '{"id": 123, "fname": "joe", "lname": "allen", "country_code": "US", "country_name": "united states", "gender": "M"}'
+
+    output_file = tmpdir.join("test_stdin_to_stdout.txt")
+    run_job("tests.stdin_to_stdout", f"echo '{tested_data}\n{tested_data2}\n{tested_data3}'", output_file)
```

Comment:
why f-string is needed? it's already a string...

Reply:
fixed

Reply:
it has interpolation of tested data

Reply:
yes, my mistake.